backend/mailer.js

```diff
@@ -6,17 +6,28 @@ var jade = require('jade');
 var hostname =process.env.HOSTNAME || 'yaydoc.herokuapp.com';
 var validation = require('../public/scripts/validation.js');
 
-const options = {
-  service: 'SendGrid',
-  auth: {
-    api_user: process.env.SENDGRID_USERNAME,
-    api_key: process.env.SENDGRID_PASSWORD
-  }
-};
+var client;
+if (process.env.SMTP === "true") {
+  client = nodemailer.createTransport({
+    host: process.env.SMTP_HOST,
```

Comment:
can we not use sendgrid like this?

Reply:
I don't think so, sendgrid need api keys. but then here we sending using smtp username and password.

Reply:
Think harder. https://sendgrid.com/blog/sending-email-nodemailer-sendgrid/